resources/com.sap.piper/pipeline/cloudSdkStageDefaults.yml

```diff
@@ -2,7 +2,17 @@ stages:
   build:
     stepConditions:
       mavenExecuteStaticCodeChecks:
-        filePattern: 'pom.xml'
```

Comment:
The step execution should still depend on the pom.xml file existence. Not sure if this can be reflected with the conditions "as is". In any case, for Java or MTA projects with pom.xml, this step should be executed even without any project specific config.

Reply:
Yeah I see your point, but if I understood the code correctly the step will not be executed if it's not configured. At least that's what [these two lines of code](https://github.com/SAP/jenkins-library/blob/master/cmd/mavenExecuteStaticCodeChecks.go#L25-L26) say right?

Reply:
But since there is the `or` relation between the condition I could also simply add this condition again to make sure its executed even if its not configured.